### DISCUSSION TITLE
issue Building Kafka Exporter Binary - Read-Only File System Error

### DISCUSSION THREAD

**Discussion by sanka-ganesh** [strimzi/strimzi-kafka-operator#10989] [Q&A]
Hi Team,

I’m encountering an issue while building the Kafka Exporter binary. After attempting to build using the `Makefile` and `kafka_exporter.go`, I receive the following error:

**go: go: could not create module cache: mkdir /app/vbuild/RHEL8-x86_64/go/1.22.3/bin/pkg: read-only file system**

It appears to be related to permissions or the environment setup, specifically with creating the module cache in the file system. Could you please assist me in resolving this issue?

Thank you in advance!

**Comment by scholzj** [comment]
Strimzi does not build any Kafka Exporter binaries. We use the binaries provided by the Kafka Exporter project. If you have any troubles building Kafka Exporter, you should probably ask there?
